salt/modules/win_service.py

```diff
@@ -462,6 +608,15 @@ def start(name, timeout=90):
 
             .. versionadded:: 2017.7.9,2018.3.4
 
+        with_deps (bool):
+            If enabled start the given service and the services the current
+            service depends on.
+
+        with_customers (bool):
+            If enabled and in case other running services depend on the to be start
+            service, this flag indicates that those other services will be started
+            as well.
```

Comment:
Perhaps it could be named `with_parents`:
```
If other running services depend on the service being started, ``True`` will start the parent services as well. Default is ``False``
```

Reply:
I changed the wording from `customer` to `parent`. I agree that this may better describe the hierarchies we see here, however it does not describe the semantics properly as well. Parents do not depend on their children necessarily, its more the way around, but this dependency exists here.

Reply:
Yeah, it was just an option... I'm fine either way.